services/supervisor/src/errors.rs

```diff
@@ -14,29 +14,24 @@
 
 use exonum_derive::*;
 
-/// Common errors emitted by transactions during execution.
-///
-/// Errors are divided into sub-groups by the corresponding error codes ranges:
-/// - 0 - 15: Common `Supervisor` errors.
-/// - 16 - 31: Errors related to artifacts.
-/// - 32 - 47: Errors related to service instances.
-/// - 48 - 64: Errors related to configuration changes.
+/// General errors that can occur within supervisor interaction.
+/// Error codes 0-15.
 #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
 #[derive(ExecutionFail)]
-pub enum Error {
-    // # General errors that can occur within supervisor interaction.
-    // Error codes 0-31.
-    // ------------------
+pub enum CommonError {
     /// Transaction author is not a validator.
     UnknownAuthor = 0,
```

Comment:
Nit: Is this error variant substantially different from `UnauthorizedCaller` in the core `CommonError`?